modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -1051,6 +1051,63 @@ definition {
 		}
 	}
 
+	@description = "LPS-164948 - Verify that the Label, Output and Formula Builder can be edited after the System Object be saved"
+	@priority = 4
+	test CanEditFormulaBuilderOnSystemObject {
+		property custom.properties = "feature.flag.LPS-164948=true";
+
+		task ("Given: A System Object with a formula field") {
+			ObjectAdmin.openObjectAdmin();
+
+			ObjectPortlet.selectSystemObject(label = "Commerce Order");
+
+			ObjectAdmin.goToFieldsTab();
+
+			ObjectAdmin.addObjectFieldViaUI(
+				fieldLabel = "Custom Formula Field",
+				fieldType = "Formula",
+				formulaFieldOption = "Decimal");
+		}
+
+		task ("When: The Formula Field is edited (Only Label, Output, and Formula Builder)") {
+			ObjectAdmin.goToFieldsDetails(label = "Custom Formula Field");
+
+			ObjectField.typeLabel(content = "Custom Formula Field Update");
+
+			Click(
+				key_filterLabel = "Output",
+				locator1 = "ObjectCustomViews#SELECT_FILTER_GENERIC");
+
+			Click(
+				key_dropdownItem = "Integer",
+				locator1 = "ObjectField#SELECT_FORMULA_FIELD_OPTION");
+
+			Type(
+				key_labelName = "Formula Builder",
+				locator1 = "TextInput#SPECIFIC_LABEL_TEXT_INPUT",
+				value1 = "field_name / field_name2");
+
+			Button.clickSave();
+		}
+
+		task ("Then: The Formula fields have been edited") {
+			ObjectAdmin.assertObjectField(
+				fieldLabel = "Custom Formula Field Update",
+				fieldType = "Formula");
+
+			ObjectAdmin.goToFieldsDetails(label = "Custom Formula Field Update");
```

Comment:
This step is not necessary:

```
ObjectAdmin.assertObjectField(
     fieldLabel = "Custom Formula Field Update",
     fieldType = "Formula");
```

Because the next step could guarantee that it's present on the field list.

```
ObjectAdmin.goToFieldsDetails(label = "Custom Formula Field Update");
```